tests/e2e/scale/noobaa/test_perf_mcg.py

```diff
@@ -0,0 +1,125 @@
+import logging
+import pytest
+import time
+
+from ocs_ci.framework.testlib import E2ETest, scale
+from ocs_ci.framework.pytest_customization.marks import vsphere_platform_required
+from ocs_ci.ocs import hsbench
+from ocs_ci.ocs.mcg_workload import get_pod_logs
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="function")
+def s3bench(request):
+    # Create hs s3 benchmark
+    s3bench = hsbench.HsBench()
+    s3bench.create_resource_hsbench()
+    s3bench.install_hsbench()
+
+    def teardown():
+        s3bench.cleanup()
+
+    request.addfinalizer(teardown)
+    return s3bench
+
+
+@scale
```

Comment:
shouldn't it be `@performance`?

Reply:
Yes, this will be changed to performance when analyzing results is completed.